src/style.css

```diff
@@ -0,0 +1,28 @@
+.starts {
```

Comment:
I think it is a typo ;)
```suggestion
.stars {
```